src/service/ConfigService.ts

```diff
@@ -52,6 +52,18 @@ export enum Preset {
     mainnet = 'mainnet',
 }
 
+export enum Assembly {
+    api = 'api',
+    demo = 'demo',
+    dual = 'dual',
+    multinode = 'multinode',
+    peer = 'peer',
+}
+
+export const defaultAssembly: Record<string, Assembly> = {
+    [Preset.bootstrap]: Assembly.multinode,
+};
+
```

Comment:
why is multinode default, won't most users of bootstrap be using it for mainnet dual?

Reply:
one machine running multiple nodes internally is the default -p bootstrap for private local networks. The config has been moved from bootstrap preset's network.yml to the new multinode assembly.